apps/dashboard/app/views/apps/_app_list_item.html.erb

```diff
@@ -3,12 +3,12 @@
     link_to(
       link.url.to_s,
       class: "list-group-item",
+      title: link.title,
       data: {
         toggle: "popover",
         content: manifest_markdown(link.description),
         html: true,
         trigger: "hover",
-        title: link.title,
```

Comment:
did you confirm popovers still work with the title? Otherwise maybe we need both title and data-title

Reply:
Good catch, but yea bootstrap respects `title` as the main option. Even the docs say for data-title: 'Default title value if title attribute isn't present.'

Reply:
Which is to say, yes, the title still correctly shows in the popover.